src/Utils/LocalizeVector.js

```diff
@@ -0,0 +1,14 @@
+const from = undefined,
+      skipUpdate = true;
+
+var globalPoint = new PIXI.Point();
+
+function localizeVector(localDisplayObject, globalVector, localVector) {
```

Comment:
localVector is optional
If provided, it will be used to store the result so no new Point is created